content/en/docs/Reference/build_raw_images_with_qemu.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Build Raw images with QEMU"
+linkTitle: "Build Raw images with QEMU"
+weight: 5
+description: This article shows how to bring your own image with Kairos, and build a Kairos derivative from scratch using base container images from popular distributions such as Ubuntu, Fedora, openSUSE, etc.
+---
+
+This page provides a reference guide on how to build raw images for Kairos using QEMU. It covers the process of using a default cloud configuration and a script to generate bootable images. The cloud configuration can be customized to suit different use cases.
+
+## Requirements
+
+- `qemu`
+- A Kairos ISO (or a custom built one)
+
```

Comment:
I'd put all this also in an alert with an info type

Reply:
:+1: done in https://github.com/kairos-io/kairos-docs/pull/355/commits/ec28350769764822ab7c253a7d74d46a8df9f159